pull-request.sh

```diff
@@ -56,9 +57,14 @@ create_pull_request() {
     REVIEWERS="$(echo -n "${8}" | jq --raw-input --slurp ".")"
     TEAM_REVIEWERS="$(echo -n "${9}" | jq --raw-input --slurp ".")"
 
+    RETVAL=0
+
     # Check if the branch already has a pull request open
     DATA="{\"base\":${TARGET}, \"head\":${SOURCE}, \"body\":${BODY}}"
-    RESPONSE=$(curl -sSL -H "${AUTH_HEADER}" -H "${HEADER}" --user "${GITHUB_ACTOR}" -X GET --data "${DATA}" ${PULLS_URL})
+    if ! RESPONSE=$(curl_wrapper -X GET --data "${DATA}" ${PULLS_URL}); then
+        RETVAL=$?
+        abort_if_fail "Error ${?} getting open PRs: ${RESPONSE}"
```

Comment:
If you grab the RETVAL here, do you want to use it in the next line instead of `${?}`